wpimath/src/main/java/edu/wpi/first/math/trajectory/TrapezoidProfile.java

```diff
@@ -67,6 +70,11 @@ public Constraints(double maxVelocity, double maxAcceleration) {
       this.maxAcceleration = maxAcceleration;
       MathSharedStore.reportUsage(MathUsageId.kTrajectory_TrapezoidProfile, 1);
     }
+
+    public <U extends Unit<U>> Constraints(
+        Measure<Velocity<U>> maxVelocity, Measure<Velocity<Velocity<U>>> maxAcceleration) {
+      this(maxVelocity.baseUnitMagnitude(), maxAcceleration.baseUnitMagnitude());
```

Comment:
Noting that this might behave unexpectedly for angular units, since the base angle unit is rotations/revolutions and not radians like most wpimath functions require for angular values. 

@calcmogul do you think we should change the units library to make radians the base unit?

Reply:
SI units should always be the base unit.